examples/Java/feature-management/README.md

```diff
@@ -0,0 +1,49 @@
+# Feature Management Sample
+
+This sample shows you how to create feature flags and integrate feature filters in a Java application using Azure App Configuration.
+
+## Connect to App Configuration Store
+
+1. In the Azure portal, navigate to your App Configuration store and select **Settings** > **Access keys** from the sidebar. Select the Read-only keys tab and copy the value of the primary connection string.
+1. Add the primary connection string as an environment variable using the variable name `APP_CONFIGURATION_CONNECTION_STRING`.
+
+    ```cmd
+    setx ConnectionString "connection-string-of-your-app-configuration-store"
+    ```
```

Comment:
Maybe include the commands for powershell and console too. This is what is usually in docs: 
 Set an environment variable named **ConnectionString**, and set it to the access key to your App Configuration store. At the command line, run the following command:

    ```cmd
    setx ConnectionString "connection-string-of-your-app-configuration-store"
    ```

    If you use Windows PowerShell, run the following command:

    ```azurepowershell
    $Env:ConnectionString = "connection-string-of-your-app-configuration-store"
    ```

    If you use macOS or Linux, run the following command:

    ```console
    export ConnectionString='connection-string-of-your-app-configuration-store'
    ```